modules/microshift-updates-rhde-config-rhel-repos.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+//microshift_updating/microshift-update-options.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-updates-rhde-config-rhel-repos_{context}"]
+= Keeping {microshift-short} and {op-system-base} in a supported configuration
+
+When using RPM updates, avoid creating an unsupported configuration or breaking your cluster by carefully managing your {op-system-base} repositories.
+
+.Repository management steps
+
+* Avoid unintended updates by locking your operating system version with the following command:
```

Comment:
Maybe get the "by using the following command" wording throughout the procedure:
```suggestion
* Lock your operating system version to avoid unintended updates by using the following command:
```